### ISSUE TITLE
Make death more unpleasant

### ISSUE THREAD

**Issue by AdamElTablawy** [yogstation13/yogstation#440]
Dying should suck ass but you can usually just get defibbed or cloned ez pz.

Changes below-

DEFIBS
1. Reduce resucitation time to 3 minutes after death (if you are smart you can just do a heart transplant and get around it) and body damage limit to greater than 100 brute or burn damage failing to resucitate
2. Make defibbing require no exosuit or jumpsuit be on
3. Make defib-resucitations have nearsightedness and braindamage.

CLONING
1. Make cloner require biomass to produce a new clone-can be gotten from bodies, meat, botany products.
2. Make clones come out with negative mutations such as blindness, deafness, or clumsy.

Death is now something to be avoided instead of shitty ez reviverino.

**Comment by timkoster1** [comment]
The effects are interesting, and so is the cloning, but changing the damage limit and settting the timer to 3 minutes? That goes a little far

**Comment by FluffySurvivor** [comment]
Because more people whining that they died is exactly what we need!

**Comment by Alek2ander** [comment]
show ads after death

**Comment by SeikoTheSeal** [comment]
>show ads after death

**Comment by AdamElTablawy** [comment]
^

**Comment by Bekons** [comment]
Make people go threw a minigame to be cloned. If they fail, there body is gibbed and they're banned for a day.

What i'm saying, is you actually want to nerf letting people play the round.

**Comment by AdamElTablawy** [comment]
I want people to fear death because as is everyone runs aroune willy nilly disarming ops of eswords knowing if they die they can just become 90 million ghost roles

**Comment by AdamElTablawy** [comment]
Also cloning is supposed to give you disabilities unupgraded but it got commented out sometime ago

**Comment by Super3222** [comment]
I've always voted for more punishments with clones.

**Comment by Paroclomas** [comment]
how about permadeath and remove the whole cloning thingie altogether.

**Comment by Paroclomas** [comment]
its true what you say about ghost roles adams, i actually prefer free golems to the actual gameplay, and i think a few people think like me. Maybe you could make that free golems are only electible from round beggining so that people who want that wont go antag fishing

**Comment by Robertruler77** [comment]
I'm firmly in the 'No' camp. Though maybe being roboticist would be more active, i doubt medical will ever be able to effectively heart transplant anywhere enough people. You do this and people will fuck off to lavaland and pAIs etc. faster.

**Comment by AdamElTablawy** [comment]
Heart transplants are already in for defibs and such and defibs are too overpowered and easy compared to cloning and need a nerf of some sort.

So @Robertruler77 come up with a less severe nerf than the over-severe nerf I have proposed here for defibs or i'll code this myself because defibs need a nerf in some form for being easy peasy death avoidance.

**Comment by Kmc2000** [comment]
Maybe make cloning require a high medical research level, so it has to be built?

That way you have to act fast to resucitate people or they're dead forever or until a roboticist borgs them

**Comment by Robertruler77** [comment]
@AdamElTablawy Unless I just get unlucky maybe 1 out of 5 people get defibbed successfully by my count. I throw a heck of a lot more corpses through cloning successfully then Defibbing them. 

Random idea fire. Say that good paddles have the old time and weak ones have the new time. Maybe limit the number of defibs or make a much weaker one for non-Parameds or CMOs, or in vice versa make the CMO and one regular set decent, restrict the paramedic ones to be weaker, have a small station with a set of paddles in medical that are even weaker, but make paramedic specific ones orderable through cargo for 2000 points a pop, with good ones at 3500 plus?

From where I sit, they're balanced. Just good enough on the basis we have med belts. Its not a no-brainer take for me; Unlike an emag for a traitor. Might as well drag the bodies back to cloning; Most are husks.

In short, Still against nerf, but maybe limit the 'Good' Ones numbers and make them costly to purchase?

Nearsightedness? Fine. Brain Damage? As long as its meme braindamage, I guess it works.

Cloning nerfs... I'm game for 1, but 2... that should be early release downsides, because full cloning shouldcome out with one that isn't all that game changing, but at 40%, as people love to do to fast process, turn you into a genetic mess, with a chance to mutate into a genetic horror? Again, random idea fire.

**Comment by darkwahn** [comment]
Not a fan of the idea, I don't think revival needs any sort of nerfs tbh.

**Comment by Stealthkibbler** [comment]
I like the biomass idea, means chef becomes more useful since meat would contain the most biomass, ever since we got self sustaining races, chef has become quite undesirable to play, Plant people feed off light, robo people eat batteries, biomass I think would be a pretty good addition and it gives chemistry the option to synthesize meat too.

**Comment by tromacodes** [comment]
Ephemeralis tried all this stuff and medical was found to be too incompetent to cope with complicated medical problems on average and that the community (at least at that time) in majority didn't want big penalties for death.

 We are also having problems with people staying with their bodies to clone. Many people would rather ghost and wait for special roles as things are now.
